### DISCUSSION TITLE
Рекорд битвы экстрасенсов

### DISCUSSION THREAD

**Discussion by audetv** [terratensor/svodd#201] [Show and tell]
Ещё только половина дня, но уже установлен рекорд по посещению.

<details><summary>Метрика</summary>
<p>

![2023-06-09_15-06-47](https://github.com/audetv/fct-search/assets/10896447/45aa7912-4be4-448f-8f64-dafa5323ad21)

</p>
</details> 

<details><summary>Статистика переходов по ссылке</summary>
<p>

![2023-06-09_15-07-41](https://github.com/audetv/fct-search/assets/10896447/2b8d24cb-a2d0-4785-b602-9989d7aad67b)

</p>
</details> 

Я удивлен, последние две недели был спад по посещению, набиралось максимум 15 чел в сутки, 9-10 чел. в день норма, по коротким ссылкам не ходили, а тут такой интерес. Минутка юмора)

**Comment by iprst** [comment]
Многих видимо заинтересовали смешные комментарии, которые были объявлены под ссылкой. 

Спад по посещению: 
- само обсуждение замедлилось;
- публиковалось меньше ссылок;
- началось лето, возникли летние дела, и тд.
